packages/ui/src/Table.tsx

```diff
@@ -36,28 +46,49 @@ export const Table = ({
     width = "100%";
   }
 
+  const shouldPaginate = true;
```

Comment:
This is just always true?

Reply:
Heh leftover from testing, thanks! Fixed.